docs: add how to compile for arm to readme (#10811)
evcc-io/evcc: README.md

./evcc
```

### Cross Compile

> To compile a version for an arm device like a raspberry pi, set the GOARCH and GOARM variables as needed, eg:

Reviewer: ```suggestion
To compile a version for an ARM device like a Raspberry Pi set GO command variables as needed, eg:
```

```sh

> GOARCH=arm GOARM=6 make

Reviewer: ```suggestion
GOOS=linux GOARCH=arm GOARM=6 make
```
